### ISSUE TITLE
Disable arbitrary tag names

### ISSUE THREAD

**Issue by tomasz13nocon** [aca/emmet-ls#55]
Right now `asdf` expands into `<asdf></asdf>`.
Is there a way to disable that and only expand valid HTML tags?

**Comment by asxraj** [comment]
Did you find a way to disable it? I'm reading the docs to try to figure it out, if you haven't I'll type my solution here

**Comment by tomasz13nocon** [comment]
No, I have not.
I'd love to see your solution.

**Comment by olrtg** [comment]
Hey! I'll see what I can do this weekend.

**Comment by Cryxto** [comment]
> Hey! I'll see what I can do this weekend.
i will gladly wait

**Comment by Cryxto** [comment]
Yes, please. This things is kinda anoying. Btw thx for the contribution @olrtg

**Comment by wassimk** [comment]
I just started using Emmet and am running into this as well. I understand it's normal behavior, but some find having a single completion snippet for anything you type is noisy. VS Code only does this for anything that starts with a capital letter, assuming it's a component name. For reference, here is a function they made to stop potentially noisy behavior:

https://github.com/microsoft/vscode-emmet-helper/commit/1359d5d4d7f14c47821074cc0d9b0eb0d15afb7b#diff-6ba93ba22e5dca6759dd4e914b378f91a6fb1e50ef4d713e4a0569c2b879ac01L309-L334

**Comment by olrtg** [comment]
I've tried to fix this but it was a lot more complicated than I thought but ended up building https://github.com/olrtg/emmet-ls which fixes this problem and for sure improves other things since it reuses vscode's integration with emmet.

**Comment by sebalfaro** [comment]
Hi @olrtg, i'm stucked with the same error, can i ask you how did you build [https://github.com/olrtg/emmet-ls](https://github.com/olrtg/emmet-ls )in order to fix the error?

**Comment by olrtg** [comment]
@sebalfaro that's a whole separate project, another language server. Go to the docs to see how you can use it!
